docs/conf.py

```diff
@@ -60,10 +61,11 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = "0.1"
 # The full version, including alpha/beta/rc tags.
-release = "0.1"
+release = get_distribution('cascade').version
+# for example take major/minor
```

Comment:
Not sure these comments are necessary. Or should they be more appropriate to this application?